### ISSUE TITLE
📌 Issue 3: Gradient Boosting

### ISSUE THREAD

**Issue by JinyunKim** [JinyunKim/timeseries-forecasting-study#4]
- [ ]  notebooks/03_gradient_boosting.ipynb 생성
- [ ]  XGBoost, LightGBM을 사용한 시계열 예측 실험
